common/libs/framing/frame_detector.h

```diff
@@ -0,0 +1,129 @@
+#ifndef __FRAME_DETECTOR__
```

Comment:
The include guard should be FRAME_DETECTOR_H_

Reply:
Done